params/protocol_params.go

```diff
@@ -90,6 +90,7 @@ const (
 	SloadGasFrontier             uint64 = 50
 	SloadGasEIP150               uint64 = 200
 	SloadGasEIP1884              uint64 = 800  // Cost of SLOAD after EIP 1884 (part of Istanbul)
+    SloadGasEIP2200              uint64 = 800  // Cost of SLOAD after EIP 2200 (part of Istanbul)
```

Comment:
Pls fix formatting, otherwise linter will whine :D

Reply:
Ping @ChainSafeSystems

Reply:
Ping @GregTheGreek

Reply:
Sorry on it! Spent the weekend without a computer... Crazy I know.

Reply:
done :)